Woche6/sentimental-credit/credit.py

```diff
@@ -0,0 +1,80 @@
+from cs50 import get_int
+
+
+def input_number(message):
+    while True:
+        try:
+            user_input = get_int(message)
+
+            if user_input <= 1:
+                print("Invalid")
+                return
+
+        except ValueError:
+            continue
+        else:
+            return user_input
+
+
+separate_digits = 0
+count = 0
+divisor = 10
+
+
+credit_card_number = input_number("Credit Number: ")
+
+workingCC = credit_card_number
+
+while workingCC > 0:
+    last_digit = workingCC % 10
+    separate_digits = separate_digits + last_digit
+    workingCC = workingCC // 100
+
+
+workingCC = credit_card_number // 10
```

Comment:
```suggestion
workingCC = credit_card_number // divisor
```
If you defined a variable, do not just use it sometimes, but everytime you can